src/lib/translations.json

```diff
@@ -30,6 +30,10 @@
       "key": "styleguide/CommentComposer/wait",
       "value": "Sie können {time} wieder einen Beitrag schreiben."
     },
+    {
+      "key": "styleguide/CommentComposer/formatting/asterisk",
+      "value": "Damit System Gendersternchen erkennt, stellen Sie einen Backslash voran: Leser\\*in"
```

Comment:
I feel like this should say `Damit System Sternchen ...` as the star is also used in other contexts for example for footnotes, date of birth, placeholders and multiplication.

Reply:
I believe, attempt here is to support those being confused when * wrapped text in comment composer becomes cursive font instead of a gender astrisk. It is a very specific "helper" to a user here.

That beeing said, I don't understand just yet what you mean, @panmau, with star being used in other contexts.

@ovbm Wording better, but I slipped you a ton of typos. How about «Um Gendersternchen sichtbar zu machen, stellen Sie einen Backslash voran: \\*»

Reply:
It is probably more of an edge case but I was thinking about usages of the star like the following:

```
According to the paper "Sample Book"* of Mr X. (* 05.04.1980) the multiplication 5*10 equals 50. 

* https://example.com
```
That's why I think it would be more fitting to use the more generic word, as the check can't know how the asterisk is used.

Reply:
Got it now. We should be able to find out how frequent astricks appear as a gender bender compared to other uses.

However, after some thinking, maybe not reference gender bender at all but help with "formatting". (Basically what you ment @panmau). Therfor I suggest:

«Um * sichtbar zu machen, stellen Sie ein Backslash voran: \\*»

(Similar might come along with underscore, using it to "genderscore": Bauer_in …)